app/main_process/menu.ts

```diff
@@ -100,23 +102,40 @@ export function setMenu(window: BrowserWindow, args: MenuItemConstructorOptionsI
     menu: Menu.buildFromTemplate(template),
     accelerators,
   };
-  applyMenu(window);
+  applyMenuBar(window);
 }
 
-export function applyMenu(window: BrowserWindow): void {
+export function applyMenuBar(window: BrowserWindow): void {
   const focusedWindow = BrowserWindow.getFocusedWindow();
   if (focusedWindow?.id == window.id) {
     const menu = menuMap[focusedWindow.id];
     Menu.setApplicationMenu(menu.menu);
     unregisterAccelerators();
     Object.entries(menu.accelerators).forEach(([accelerator, uuid]) => {
       globalShortcut.register(accelerator, () => {
-        menuClick(window, uuid);
+        menuBarClick(window, uuid);
       });
     });
   }
 }
 
+export function showMenuBar(window: BrowserWindow): void {
+  menuMap[window.id].menu.popup({
+    x: 0,
+    y: 55,
```

Comment:
Can we dynamically calculate this?

Reply:
not an easy way. imo not really worth it